.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: '/'
+    schedule:
+      interval: weekly
```

Comment:
Could move this to monthly to reduce the noise

Reply:
Maybe bi-annually?  All deps are devDeps, so it's not critical to keep everything up to date.

Reply:
```suggestion
      interval: monthly
```